src/data/invalid/Database-WorkflowExecutionActivity-invalid.yaml

```diff
@@ -0,0 +1,16 @@
+activity_set:
+  #test should fail because activity_set does not exist anymore, WorkflowExecutionActivity is abstract
+  - id: nmdc:wfact-11-hemh0a87.1
+    name: WXA for nmdc:wfact-11-hemh0a87.1
+    type: nmdc:WorkflowExecutionActivity
```

Comment:
```suggestion
    type: nmdc:WorkflowExecution
```

Reply:
wow I'm surprised the example data files didn't catch that!

Reply:
maybe because we don't instantiate it? it's an abstract class?